tests/MongoDB.Driver.Tests/Communication/Security/AwsAuthenticationTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Driver.TestHelpers;
+using System;
+using Xunit;
+
+namespace MongoDB.Driver.Tests.Communication.Security
+{
+    [Trait("Category", "Authentication")]
+    [Trait("Category", "AwsMechanism")]
+    public class AwsAuthenticationTests
+    {
+        [Fact]
+        public void Aws_authentication_should_behave_as_expected()
```

Comment:
@rstam regarding Dmitry's comment https://github.com/MikalaiMazurenka/mongo-csharp-driver/pull/22/files#r393350233
I think it's not crucial for this test to be run only on AWS task, but if yes, I'm thinking about checking an environment variable, e.g. "AWS_TESTS_ENABLED".
Could you please suggest which way it should be done here?

Reply:
If we decide to do this, we could use Dima's suggestion from the OCSP PR:
`RequireEnvironment.Check().EnvironmentVariable("name-here");`

Reply:
Is it ECS container test?
I mean your comment here:
![image](https://user-images.githubusercontent.com/44870738/77159836-2cb76f80-6ab7-11ea-898d-abd58820dc06.png)

Reply:
I like Vincent's suggestion:

    RequireEnvironment.Check().EnvironmentVariable("AWS_TESTS_ENABLED");

Reply:
Done.

> Is it ECS container test?

It's AWS con. string, env. variables and EC2 test. ECS is out of scope for .NET